pkg/actors/actors.go

```diff
@@ -75,6 +75,7 @@ type actorsRuntime struct {
 	activeTimers        *sync.Map
```

Comment:
timer would have similar issue, wouldn't it? If yes, then we would need to fix timer after this pr.

Reply:
Will check and file a separate issue